tests/python_tests/test_whisper_generate_api.py

```diff
@@ -199,7 +199,7 @@ def test_whisper_constructors(model_descr, test_sample):
     expected = opt_pipe(test_sample)["text"]
 
     genai_result = ov_genai.WhisperPipeline(
-        str(path), device="CPU", config={"ENABLE_MMAP": False}
+        str(path), device="CPU", **{"ENABLE_MMAP": False}
```

Comment:
is it possible to use `ov.props.enable_mmap = True` ?

See https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_runtime/test_properties.py#L268

Reply:
No, currently it won't work. We need to write `ov_genai.WhisperPipeline(ENABLE_MMAP=True)`
Should I convert property names to uppercase? 
Other option is to list all possible property names in this method: https://github.com/openvinotoolkit/openvino.genai/blob/b85991ea85717ab9fd856d723bd4716011844059/src/python/py_utils.cpp#L156

Reply:
No, works only like this:
`openvino_genai.LLMPipeline(args.model_dir, device, ENABLE_MMAP=ov.properties.enable_mmap)`

Also, the name of property should be uppercase, otherwise plugin doesn't recognize it.